sdk/cdn/Microsoft.Azure.Management.Cdn/src/Customization/Models/CookiesMatchConditionParameters.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Management.Cdn.Models
+{
+
+    public partial class CookiesMatchConditionParameters
+    {
+        /// <summary>
+        /// Initializes a new instance of the CookiesMatchConditionParameters
+        /// class.
+        /// </summary>
+        /// <param name="selector">Name of Cookies to be matched</param>
+        /// <param name="operatorProperty">Describes operator to be matched.
+        /// Possible values include: 'Any', 'Equal', 'Contains', 'BeginsWith',
+        /// 'EndsWith', 'LessThan', 'LessThanOrEqual', 'GreaterThan',
+        /// 'GreaterThanOrEqual'</param>
+        /// <param name="matchValues">The match value for the condition of the
+        /// delivery rule</param>
+        /// <param name="negateCondition">Describes if this is negate condition
+        /// or not</param>
+        /// <param name="transforms">List of transforms</param>
+        public CookiesMatchConditionParameters(string selector = null, string operatorProperty = null, IList<string> matchValues = null, bool? negateCondition = null, IList<string> transforms = null)
+        {
```

Comment:
I would think we would do this here:

        public CookiesMatchConditionParameters(string operatorProperty, string selector = default(string), IList<string> matchValues = default(IList<string>), bool? negateCondition = default(bool?), IList<string> transforms = default(IList<string>)){
      this.CookiesMatchConditionParameters(operatorProperty, selector, negateCondition, matchValues, transforms);

I the this.CookiesMatchConditionParameters above should also be defined here identical to the generated one but marked internal so it is not exposed outside of this.

Reply:
Does this work with Autorest C# v2 generator? I'm able to add the new constructor because that doesn't require any change for the generated code, but the generator isn't removing the existing constructor when I replace it with an internal one in my customizations partial class. I can't find any documentation about customization using v2 of the C# generator, only for v3.